pages/index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Esther's Portfolio</title>
+    <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon">
+    <link href="/styles/style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
 </head>
+
 <body>
-  
+    <header class="header-container">
+      <h1 id="page-title">Esther's Portfolio</h1>
+      <nav class="nav">
+        <ul>
+          <li><a href="index.html">Home</a> |</li>
+          <li><a href="about.html">About Me</a> |</li>
+          <li><a href="portfolio.html">Portfolio</a> |</li>
+          <li><a href="contact.html">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      
+      <div class="intro-container">
+        <section>
+          <img id="portrait" src="/images/portrait.jpg" alt="A smiling woman">
+
+          <article class="introduction">
+            <h2>Lorem ipsum</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas lobortis, ex quis venenatis interdum, dolor libero pharetra mauris, ut mollis erat enim in odio. Ut luctus iaculis eleifend. Curabitur in convallis nunc. Aenean consectetur, dui id ultricies dictum, erat est dapibus magna, in porta risus nisi at massa. Donec rutrum augue et varius bibendum. Mauris in sapien risus. Vestibulum interdum sapien sed scelerisque laoreet. Vivamus sit amet ex eu sapien ornare efficitur. Integer facilisis arcu quis nulla tincidunt, vel tempus risus venenatis. Sed sagittis nisl eget justo pulvinar, sed aliquam purus aliquet. Praesent eros dolor, dignissim non sollicitudin ut, interdum in ipsum. Sed suscipit ante lorem, eu interdum diam convallis nec.</p>
+            </article>
+      </section>
+      </div>
+    </main>
+
+    <footer>
+      <div class="footer">
+        <p>© Esther Annorzie, 2022</p>
+        <a class="social-media" href="https://facebook.com/estherannorzie"><p>Facebook</p></a>
+        <a href="https://linkedin.com/in/estherannorzie"><p>LinkedIn</p></a>  
+        <a href="https://github.com/estherannorzie"><p>GitHub</p></a>
```

Comment:
It isn't necessary, but since this is conceptually a list of the ways to contact you, I would consider placing the contact links in a `ul` and styling it as necessary to show the links side by side.